.editorconfig

```diff
@@ -297,7 +297,7 @@ dotnet_diagnostic.CA1823.severity = warning
 # Avoid zero-length array allocations.
 dotnet_diagnostic.CA1825.severity = warning
 # Do not use Enumerable methods on indexable collections. Instead use the collection directly
-dotnet_diagnostic.CA1826.severity = warning
+dotnet_diagnostic.CA1826.severity = none
```

Comment:
Disabled this warning as there are specific situations in which you want to use `.FirstOrDefault()` since you cannot guarantee the collection has any elements to access directly without manual checks/extra boilerplate code.